### ISSUE TITLE
Youtube Live causing lag due to ublock filtering. 

### ISSUE THREAD

**Issue by OwenOblivi** [gorhill/uBlock#2711]
I noticed this issue recently because my YouTube live players keep getting out of sync, so I started to look into the issue.

With ublock enabled I got these errors:
[![https://gyazo.com/851fdeb24eb5d9039e51a0ac58fa9b54](https://gyazo.com/851fdeb24eb5d9039e51a0ac58fa9b54/raw)](https://gyazo.com/851fdeb24eb5d9039e51a0ac58fa9b54)

with out ublock its pretty clean: 
[![https://gyazo.com/d6b2647560f590bb8975f72b3db2b6c1](https://gyazo.com/d6b2647560f590bb8975f72b3db2b6c1/raw)](https://gyazo.com/d6b2647560f590bb8975f72b3db2b6c1)

I've tested this both on youtube and my favorite streamers site Iceposeidon.com (he just uses an embded youtube player on his site)

Hope this helps!

**Comment by anewuser** [comment]
Those aren't "errors". They are merely notices that uBlock Origin is working exactly as expected: it blocks ads and privacy-invading URL requests.  If you can't understand that, please avoid enabling your browser developer tools. The lag is likely due to connection problems on your side or the streamer's side. Anyway, this isn't a help forum. 

You've ignored and deleted the template text that appears when you create a new issue.

That text isn't there just for decoration.  

Please close this issue and read this carefully before posting here again: [CONTRIBUTING.md](https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md).

**Comment by gorhill** [comment]
`ERR_BLOCKED_BY_CLIENT` is the browser informing you a network request was blocked by an extension, i.e. uBO is doing it's intended job.
